src/controls/orbit.rs

```diff
@@ -32,9 +32,12 @@ pub struct Builder {
 
 impl Builder {
     /// Create new `Builder` with default values.
-    pub fn new(object: &Object) -> Self {
+    pub fn new<T>(object: T) -> Self
```

Comment:
I think `T: Object` should be preferred, calling `object.upcast()` instead of `object.as_ref().clone()`. Please debate otherwise.